components/TextFields/src/MDCTextField.m

```diff
@@ -672,6 +694,19 @@ + (BOOL)requiresConstraintBasedLayout {
   return YES;
 }
 
+- (UIView *)viewForFirstBaselineLayout {
+  return self.inputLayoutStrut;
+}
+
+- (UIView *)viewForLastBaselineLayout {
+  return self.inputLayoutStrut;
+}
+
+// TODO: (larche) Remove when we drop iOS 9 support
```

Comment:
s/larche/#4390/

#4390 is the tracking bug for all the cleanup we can do post iOS 9.

Reply:
Done.